crates/bevy_ecs/src/query/iter.rs

```diff
@@ -214,6 +214,59 @@ where
 {
 }
 
+/// An iterator over `K`-sized combinations of query items without repetition.
+///
+/// In this context, a combination is an unordered subset of `K` elements.
+/// The number of combinations depend on how `K` relates to the number of entities matching the [`Query`] (called `N`):
+/// - if `K = N`, only one combination exists,
+/// - if `K < N`, there are <sub>N</sub>C<sub>K</sub> combinations (see the [performance section] of `Query`),
```

Comment:
Do you have a screenshot of how this looks in the docs? Having `<sub>` first looks wrong to me.

Reply:
It should render the same as here: <sub>N</sub>C<sub>K</sub>

Reply:
I'm not familiar with this notation. What does it mean?

Reply:
It's a common notation of the [binomial coefficient]. You can also see the rendered docs by checking out the branch and doing the cargo doc thing.

[binomial coefficient]: https://en.wikipedia.org/wiki/Binomial_coefficient#History_and_notation

Reply:
Interesting, if it's common enough I guess it's fine. Otherwise I think it would be good to switch to another notation or to describe it textually. I guess the problem is that we are constrained by the Markdown formatting so we can't use the "classical" notation.

Reply:
It's still better than to write a fraction with factorials in it.